tests/deal.II/grid_tools_halo_layer_01.output

```diff
@@ -0,0 +1,3303 @@
+0.00000 0.250000 1.00000 2 1
+0.00000 0.250000 0.750000 2 1
+
+0.00000 0.500000 0.750000 2 1
+0.250000 0.500000 0.750000 2 1
+0.250000 0.500000 1.00000 2 1
+0.00000 0.500000 1.00000 2 1
+0.00000 0.500000 0.750000 2 1
+
+0.00000 0.250000 0.750000 2 1
+0.00000 0.500000 0.750000 2 1
+
+0.250000 0.250000 0.750000 2 1
+0.250000 0.500000 0.750000 2 1
+
+0.250000 0.250000 1.00000 2 1
+0.250000 0.500000 1.00000 2 1
+
+0.00000 0.250000 1.00000 2 1
+0.00000 0.500000 1.00000 2 1
+
+0.250000 0.250000 0.750000 2 1
+0.500000 0.250000 0.750000 2 1
+0.500000 0.250000 1.00000 2 1
+0.250000 0.250000 1.00000 2 1
+0.250000 0.250000 0.750000 2 1
+
+0.250000 0.500000 0.750000 2 1
+0.500000 0.500000 0.750000 2 1
+0.500000 0.500000 1.00000 2 1
+0.250000 0.500000 1.00000 2 1
+0.250000 0.500000 0.750000 2 1
+
+0.250000 0.250000 0.750000 2 1
+0.250000 0.500000 0.750000 2 1
+
+0.500000 0.250000 0.750000 2 1
+0.500000 0.500000 0.750000 2 1
+
+0.500000 0.250000 1.00000 2 1
+0.500000 0.500000 1.00000 2 1
+
+0.250000 0.250000 1.00000 2 1
+0.250000 0.500000 1.00000 2 1
+
+0.500000 0.00000 0.500000 2 3
+0.750000 0.00000 0.500000 2 3
+0.750000 0.00000 0.750000 2 3
+0.500000 0.00000 0.750000 2 3
+0.500000 0.00000 0.500000 2 3
+
+0.500000 0.250000 0.500000 2 3
+0.750000 0.250000 0.500000 2 3
+0.750000 0.250000 0.750000 2 3
+0.500000 0.250000 0.750000 2 3
+0.500000 0.250000 0.500000 2 3
+
+0.500000 0.00000 0.500000 2 3
+0.500000 0.250000 0.500000 2 3
+
+0.750000 0.00000 0.500000 2 3
+0.750000 0.250000 0.500000 2 3
+
+0.750000 0.00000 0.750000 2 3
+0.750000 0.250000 0.750000 2 3
+
+0.500000 0.00000 0.750000 2 3
+0.500000 0.250000 0.750000 2 3
+
+0.750000 0.00000 0.500000 2 1
+1.00000 0.00000 0.500000 2 1
+1.00000 0.00000 0.750000 2 1
+0.750000 0.00000 0.750000 2 1
+0.750000 0.00000 0.500000 2 1
+
+0.750000 0.250000 0.500000 2 1
+1.00000 0.250000 0.500000 2 1
+1.00000 0.250000 0.750000 2 1
+0.750000 0.250000 0.750000 2 1
+0.750000 0.250000 0.500000 2 1
+
+0.750000 0.00000 0.500000 2 1
+0.750000 0.250000 0.500000 2 1
+
+1.00000 0.00000 0.500000 2 1
+1.00000 0.250000 0.500000 2 1
+
+1.00000 0.00000 0.750000 2 1
+1.00000 0.250000 0.750000 2 1
+
+0.750000 0.00000 0.750000 2 1
+0.750000 0.250000 0.750000 2 1
+
+0.500000 0.250000 0.500000 2 3
+0.750000 0.250000 0.500000 2 3
+0.750000 0.250000 0.750000 2 3
+0.500000 0.250000 0.750000 2 3
+0.500000 0.250000 0.500000 2 3
+
+0.500000 0.500000 0.500000 2 3
+0.750000 0.500000 0.500000 2 3
+0.750000 0.500000 0.750000 2 3
+0.500000 0.500000 0.750000 2 3
+0.500000 0.500000 0.500000 2 3
+
+0.500000 0.250000 0.500000 2 3
+0.500000 0.500000 0.500000 2 3
+
+0.750000 0.250000 0.500000 2 3
+0.750000 0.500000 0.500000 2 3
+
+0.750000 0.250000 0.750000 2 3
+0.750000 0.500000 0.750000 2 3
+
+0.500000 0.250000 0.750000 2 3
+0.500000 0.500000 0.750000 2 3
+
+0.750000 0.250000 0.500000 2 1
+1.00000 0.250000 0.500000 2 1
+1.00000 0.250000 0.750000 2 1
+0.750000 0.250000 0.750000 2 1
+0.750000 0.250000 0.500000 2 1
+
+0.750000 0.500000 0.500000 
```

Comment:
On 08/24/2015 08:33 AM, jppelteret wrote:
> I'll try another output format instead of Gnuplot.
It's not about the data format, but about the amount of data. Maybe a 
smaller mesh? Maybe also just reduce the number of zeros in the output. 
Or reports something different altogether: e.g., loop over all cells and 
simply output the material id for each cell? You're not concerned about 
the positions of vertices anyway, nor about actually plotting the data. 
In any case, I think it should be possible to do what you want to test 
with less than 60kB of output ;-)